subjects/ascii-art-web/audit/README.md

```diff
@@ -1,6 +1,8 @@
 #### Functional
 
-###### Has the requirement for the allowed packages been respected? (Reminder for this project: (only [standard packages](https://golang.org/pkg/)
+###### Has the requirement for the allowed packages been respected? (only the [standard go](https://golang.org/pkg/) packages are allowed)
+
+###### Does the code obey the [good practices](https://public.01-edu.org/subjects/good-practices/)?
 
```

Comment:
For this one i'm not sure if it is a good idea to put it as a required question. It is good to obey the good-practices but most of the times a big project must have a good structure and practices for it to work properly. So i think its better to leave it as a bonus question